src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -81,6 +81,10 @@ public function find(bool $includeArgs = true): string|false
             $dirs[] = 'C:\xampp\php\\';
         }
 
+        if ($herdPath = getenv('HERD_HOME')) {
+            $dirs[] = $herdPath . ('\\' === \DIRECTORY_SEPARATOR ? '\\bin' : '/bin');
```

Comment:
```suggestion
            $dirs[] = $herdPath . \DIRECTORY_SEPARATOR . 'bin';
```

This is much simpler

Reply:
Yeah of course 🤦 
Thank you!